types/aws-lambda/index.d.ts

```diff
@@ -190,6 +197,7 @@ interface ProxyResult {
  */
 export type Handler = (event: any, context: Context, callback?: Callback) => void;
 export type ProxyHandler = (event: APIGatewayEvent, context: Context, callback?: ProxyCallback) => void;
+export type CustomAuthorizerHandler = (event: CustomAuthorizerEvent, context: Context, callback?: Callback) => void;
```

Comment:
I did the addition of "CustomAuthorizerCallback" and "AuthResponse".

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/15387/commits/60bda6cf4f8cd130f8a47a3dc3cdcd71bbffc6ea